src/app/drive/components/DriveExplorer/DriveExplorerItem/DriveExplorerListItem/DriveExplorerListItem.tsx

```diff
@@ -96,7 +96,8 @@ const DriveExplorerListItem = ({ item }: DriveExplorerItemProps): JSX.Element =>
             title={items.getItemDisplayName(item)}
             onClick={onNameClicked}
           >
-            {items.getItemDisplayName(item)}
+            {/* TODO: CHANGE getItemDisplayName OF internxt/lib IN ORDER TO USE plain_name*/}
+            {item.plain_name ?? item.name}
```

Comment:
Given we want this change to be silent yet, we want to store the file name unencrypted in the backend but not yet use it to show it in the front end.

Since we have to first unencrypt the existing file names, and we haven't done that yet.

So for the moment anything that is to display name, should still work the way it is. We just want to send the file name unencrypted.

Reply:
Perfect, I'll change it back to the way it was!